tests/precalculations/test_WordPredictionPrecalculation.py

```diff
@@ -0,0 +1,70 @@
+from unittest import TestCase
+
+import numpy as np
+from pandas import DataFrame
+
+from shift_detector.precalculations.Store import Store
+from shift_detector.precalculations.WordPredictionPrecalculation import WordPredictionPrecalculation
+
+
+class TestCreateDetector(TestCase):
+
+    def setUp(self):
+        alphabet = [chr(letter) for letter in range(ord('a'), ord('z')+1)]
+
+        col = []
```

Comment:
What happens here? Either be more descriptive with variable names or make comments ...